src/cholesky.jl

```diff
@@ -1,5 +1,5 @@
 # Generic Cholesky decomposition for fixed-size matrices, mostly unrolled
-non_hermitian_error() = throw(LinearAlgebra.PosDefException(-1))
+non_hermitian_error() = throw(L∈earAlgebra.PosDefException(-1))
```

Comment:
Let's not be too excited about replacing `in` with `∈` :smile:

Reply:
Unfortunately, because `∈` is a unary operator, I couldn't define `const L∈nearAlgebra = LinearAlgebra`. Looks like I have to change it back.